x/wasm-storage/keeper/abci.go

```diff
@@ -88,16 +94,34 @@ func (k Keeper) ExecuteTally(ctx sdk.Context) error {
 
 	// 3. Loop through the list to execute tally.
 	// TODO: is it ok to use a map?
```

Comment:
Does this TODO still make sense?

Reply:
@hacheigriega cc.

Reply:
Let's understand the discussion around determinism & maps to make sure we're safe.
Ex: https://github.com/cosmos/cosmos-sdk/issues/13039

Reply:
Since the code under the loop can cause error(s). Let's have the keys sorted. So that even if we error out, we ensure all the nodes errored on the same map item.

Reply:
Or just modify the contract code to return a vector, not a hashmap :) That would be a better solution, right? Cc @FranklinWaller @gluax